src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const cells = rows[i].cells;
+  const positionCells = cells[1];
```

Comment:
Ensure that each row has at least two cells to avoid accessing an undefined index, which would cause a runtime error.